docs_user/modules/proc_configuring-a-ceph-backend.adoc

```diff
@@ -1,40 +1,30 @@
 [id="configuring-a-ceph-backend_{context}"]
 
-= Configuring a Ceph backend
+= Configuring a {Ceph} back end
```

Comment:
@fmount  Should we be referring to the back end as the "Ceph back end" or the "Red Hat Ceph Storage back end"?

Reply:
I think downstream we should keep referring to RHCS [1], while Ceph is sufficient for the upstream doc.
Do we want to put an `if` statement to make this distinction more explicit? 

[1] https://docs.redhat.com/en/documentation/red_hat_ceph_storage/7

Reply:
We don't need the if statement. The {Ceph} attribute renders as "Ceph" upstream, while downstream it renders as "Red Hat Ceph Storage".